### PR TITLE
docs: fix code examples and statement expressions

### PR DESCRIPTION
**Pull request by RainyNight9**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/react/use-with-umi.en-US.md [hunk 1]
```diff
@@ -133,12 +133,18 @@ Create a new `src/components/ProductList.tsx` file with the following code.
 ```tsx
 import React from 'react';
 import { Button, Popconfirm, Table } from 'antd';
+import type { TableProps } from 'antd';
 
-const ProductList: React.FC<{ products: { name: string }[]; onDelete: (id: string) => void }> = ({
+interface DataType {
+    id: string;
+    name: string;
```
Comment (afc163) [inline docs/react/use-with-umi.en-US.md:140]:
```suggestion
  id: string;
  name: string;
```

**Comment by RainyNight9** [comment]
> Hi~

Hi

**Comment by afc163** [comment]
Could you check my suggestions?

**Comment by RainyNight9** [comment]
> Could you check my suggestions?

**Review by afc163 — APPROVED** [review]

**Pull request merged**